examples/visualization/eyetracking_plot_heatmap.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-eyetrack-heatmap:
+
+=============================================
+Plotting eye-tracking heatmaps in MNE-Python
+=============================================
+
+This tutorial covers plotting eye-tracking position data as a heatmap.
+
+.. seealso:: :ref:`tut-importing-eyetracking-data`
+
+"""
+
+# %%
+# Data loading
+# ------------
+#
+# As usual we start by importing the modules we need and loading some
+# :ref:`example data <eyelink-dataset>`: eye-tracking data recorded from SR research's
+# ``'.asc'`` file format.
+
+
+import matplotlib.pyplot as plt
+
+import mne
+from mne.viz.eyetracking import plot_gaze
+
+
+# define variables to pass to the plot_gaze function
+px_width, px_height = 1920, 1080
+cmap = plt.get_cmap("viridis")
```

Comment:
I would probably move these to the place where they're first used. There's cognitive overhead when reading the code to see these vars defined but then have to read a bunch of other code before seeing them used.